pkgs/development/r-modules/wrapper.nix

```diff
@@ -1,9 +1,13 @@
-{ stdenv, R, makeWrapper, recommendedPackages, packages }:
+{ stdenv, R, libcxx, makeWrapper, recommendedPackages, packages }:
 
 stdenv.mkDerivation {
   name = R.name + "-wrapper";
 
-  buildInputs = [makeWrapper R] ++ recommendedPackages ++ packages;
+  buildInputs = [makeWrapper R] ++ recommendedPackages ++ packages
+    ++ stdenv.lib.optionals stdenv.isDarwin R.darwinFrameworks;
+
+  NIX_CFLAGS_COMPILE =
+    stdenv.lib.optionalString stdenv.isDarwin "-I${libcxx}/include/c++/v1";
```

Comment:
Are you certain that those changes in the wrapper are necessary? Based on my understanding, the modifications to `buildInputs` and `NIX_CFLAGS_COMPILE` have no effect on the wrapper whatsoever, so I tend to believe that those changes can be just dropped.